packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -199,6 +200,41 @@ const DailyItemTasks: GarboTask[] = [
     },
     spendsTurn: false,
   },
+  {
+    name: "Sept-Ember",
```

Comment:
Make two different tasks, one for checking the shop and one for purchasing

Reply:
Okay